nwb_conversion_tools/nwbconverter.py

```diff
@@ -121,11 +121,15 @@ def run_conversion(
         assert (not save_to_file and nwbfile_path is None) or nwbfile is None, \
             "Either pass a nwbfile_path location with save_to_file=True, or a nwbfile object, but not both!"
 
+        # Validate conversion options
         if conversion_options is None:
             conversion_options = dict()
         else:
             validate(instance=conversion_options, schema=self.get_conversion_options_schema())
 
+        # Validate metadata
+        validate(instance=metadata, schema=self.get_metadata_schema())
```

Comment:
@luiztauffer I'm wondering now if we should actually decouple this and move this one line to a PR of its own accompanied by a checklist of each supported data format to be tested/confirmed by us, and then followed by a version bump+release. B/c just adding this here could potentially cause a lot of breakages in previous uses

Reply:
@CodyCBakerPhD makes sense to me. Given that the user can also modify metadata, it would be better to have this separated as a class method as well